src/grading-settings/scss/GradingSettings.scss

```diff
@@ -2,12 +2,18 @@
 @import "../assignment-section/AssignmentSection.scss";
 @import "../grading-scale/GradingScale.scss";
 
-.grading-label {
-  font: normal $font-weight-bold .75rem/1.25rem $font-family-base;
-  color: $gray-500;
-}
+.grading {
+  .help-sidebar {
+    margin-top: 8.75rem;
+  }
+
+  .grading-label {
+    font: normal $font-weight-bold .75rem/1.25rem $font-family-base;
+    color: $gray-500;
+  }
 
-.grading-description {
-  font: normal .75rem/1.5rem $font-family-base;
-  color: $gray-700;
+  .grading-description {
+    font: normal .75rem/1.5rem $font-family-base;
```

Comment:
use `x-small text-gray-700 font-weight-normal` instead
i think isn't going to make much of a difference